app/views/pages/resources.html.erb

```diff
@@ -1,9 +1,100 @@
+      </li>
+    </ul>
+  </div>
+  <div class="resources-links">
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <%= image_tag 'logos/military_one_source_logo.svg' %>
+      </div>
+      <div class="usa-width-two-thirds">
+        <h4><%= link_to 'Military One Source', 'https://www.militaryonesource.mil/' %></h4>
+        <p>Military One Source has a bunch of articles about how to plan your move, find housing and employement in your new location, and how to deal with the stress  caused from uprooting your family.</p>
+        <ul class="usa-unstyled-list two-column-list">
+          <li><%= link_to 'Master Your Move Guide', 'https://www.militaryonesource.mil/pcs' %></li>
+          <li><%= link_to 'Military Spouses’ Blog', 'https://blog-brigade.militaryonesource.mil/blogs/moving-2/' %></li>
+          <li><%= link_to 'Moving Benefits', 'http://www.militaryonesource.mil/web/mos/moving-benefits' %></li>
+          <li><%= link_to '24/7 Family Support Line', 'https://www.militaryonesource.mil/web/mos/confidential-help' %></li>
+          <li><%= link_to 'Helping Your Kids Cope', 'https://www.militaryonesource.mil/web/mos/-/nine-ways-to-help-your-kids-cope-with-moving' %></li>
+        </ul>
+      </div>
+    </div>
 
-  <%# TODO Add page content %>
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <%= image_tag 'logos/military_installations_logo.svg' %>
+      </div>
+      <div class="usa-width-two-thirds">
+        <h4><%= link_to 'Military Installations', 'http://www.militaryinstallations.dod.mil/' %></h4>
+        <p>Everything you need to know about your new base including school districts, pet restrictions, and on-base contact infomation.</p>
+      </div>
+    </div>
 
-  <p>Coming soon!</p>
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <%= image_tag 'logos/sesame_street_logo.svg' %>
+      </div>
+      <div class="usa-width-two-thirds">
+        <h4><%= link_to 'PBS Kids Relocation Resources', 'http://sesamestreetformilitaryfamilies.org/topic/relocation/' %></h4>
+        <p>Videos, worksheets, and tips to help prepare the youngest members of the family for the PCS process.</p>
+      </div>
+    </div>
+
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <h4>Household Goods Policies and Supplements</h4>
+      </div>
+      <div class="usa-width-two-thirds">
+        <ul class="usa-unstyled-list">
+          <li><%= link_to 'Joint Travel Regulation', 'http://www.defensetravel.dod.mil/site/travelreg.cfm' %></li>
+          <li><%= link_to 'Defense Travel Regulation', 'https://www.ustranscom.mil/dtr/' %></li>
+          <li><%= link_to 'Air Force Household Goods Supplement', 'http://static.e-publishing.af.mil/production/1/af_a4_7/publication/afi24-501/afi24-501.pdf' %></li>
+          <li><%= link_to 'Marine Corps Household Goods Supplement', 'https://www.move.mil/documents/dod/MCO_4600_39_19_AUG_2016.pdf' %></li>
```

Comment:
We'll want to account for links to what is now `www.move.mil` at some point down the line if/when the current site moves to another domain.